README.md

```diff
@@ -17,23 +28,21 @@ In order to quickly setup all components of the stack, run:
 kubectl apply -f manifests/ -R
 ```
 
-The above command sets up JupyterHub, an API for training using Tensorflow, and a set of deployment files for serivng. 
+The above command sets up JupyterHub, an API for training using Tensorflow, and a set of deployment files for serving. 
 Used together, these serve as configuration that can help a user go from training to serving using Tensorflow with minimal
 effort in a portable fashion between different environments. You can refer to the instructions for using each of these components below. 
 
 ## Setup
 
+This documentation assumes you have a Kubernetes cluster already available. For specific Kubernetes installations, additional configuration may be necessary.
+
 ### Minikube
 
 [Minikube](https://github.com/kubernetes/minikube) is a tool that makes it easy to run Kubernetes locally. Minikube runs a single-
 node Kubernetes cluster inside a VM on your laptop for users looking to try out Kubernetes or develop with it day-to-day. 
-The below steps apply to a minikube cluster - the latest version as of writing this documetation is 0.23.0. You must also have 
-kubectl configured to access minikube.
+The below steps apply to a minikube cluster - the latest version as of writing this documetation is 0.23.0. You must also have kubectl configured to access minikube.
 
-### Google Kubernetes Engine
```

Comment:
Do not delete this section. It is the best way currently to try out GPUs k8s clusters.

Reply:
That's fine, but it shouldn't be here - or if it is, it should have other solutions as well. As it stands, it makes it seem that this is the only way to create a production cluster.